Source/WebCore/platform/graphics/gbm/GraphicsContextGLGBM.cpp

```diff
@@ -83,6 +83,11 @@ void GraphicsContextGLGBM::setContextVisibility(bool)
 {
 }
 
+bool GraphicsContextGLGBM::setDrawingBufferColorSpaceIfSupported(DestinationColorSpace)
+{
+    return false;
+}
+
```

Comment:
Why implement these in each derived class from `GraphicsContextGL` instead of this being the default behaviour? Less code is better code.

Reply:
I did try that first! But I got linker errors somewhere because it couldn't find that base-class implementation! I'll try again...

Reply:
The base declaration needs to be declared as  exported.

Reply:
Oh, thanks for the tip!
I did manage to fix the link issue by providing an override in RemoteGraphicsContextGLProxy.h (that's where the "missing" base implementation was called from). So do I still need to export this one?

Reply:
In case you have a non-inline base implementation, you need to export.
In case you have inline base implementation or pure virtual function, you must not export.

There should probably not be a shared implementation here, no backend can share anything useful except empty implementation and empty implementation is not that useful.